### DISCUSSION TITLE
Server automatically crashs !!!

### DISCUSSION THREAD

**Discussion by STAROFWIND** [milvus-io/milvus#13793] [Q&A and General discussion]
Hi Everyone,
I installed Milvus by using docker-compose like the guild:  https://milvus.io/docs/v2.0.0/install_standalone-docker.md
I run the server and got successfully like this: 
![image](https://user-images.githubusercontent.com/53179529/146748128-26387b16-9308-466c-b632-cbe0c6bcd24f.png)
But after about 1 minute, The server turn off without any my operation. 
![image](https://user-images.githubusercontent.com/53179529/146748365-24039609-980b-4895-a367-91731f9f6252.png)
I only see my Memory increases to ~15G / 128G. Does It relate to "volumes" ?. I don't still have workaround for this issue. 
Any help ? Thank you so much

**Comment by yhmo** [comment]
@STAROFWIND 
I noticed you have created issue #13795, developers are investigating it. Let's use the issue to trace the bug.
